### PR TITLE
View story

### PR DESCRIPTION
**Pull request by aspensantos**
https://user-images.githubusercontent.com/35615166/111043854-f8009e80-8412-11eb-84d1-0a8c46b59a0b.mp4

### CODE REVIEW

**Review by hanlinc27 — APPROVED** [review]
It works super well for me! Small styling - when I go from Upload --> Preview, the heading text is shifted up? Going from AllStories --> Story Preview looks fine though. Screenshot below for reference.
image

File: ui/src/pages/Upload.tsx [hunk 1]
```diff
@@ -148,6 +151,63 @@ export const Upload: React.FC = () => {
...
+        longitude: 0,
+        ID: 0,
+        CreatedAt: "",
+        UpdatedAt: "",
+        DeletedAt: {
+          Time: "",
+          Valid: true,
+        },
+      };
+      console.log(storyFromData);
```
Comment (nimwijetunga) [inline ui/src/pages/Upload.tsx]: remove please

File: ui/src/pages/Upload.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import {
```
Comment (nimwijetunga) [inline ui/src/pages/Upload.tsx:1]: One thing you'll have to consider is doing the same thing for our edit story page (once that's merged in). How flexible is this logic?

File: ui/src/pages/AllStories.tsx [hunk 3]
```diff
@@ -8,6 +8,7 @@ import * as React from "react";
 import styled from "styled-components";
```
Comment (nimwijetunga) [inline ui/src/pages/AllStories.tsx:8]: I'm not entirely sure what this is doing but do you link the all stories page to this view story page here or is it something else?

Comment (nimwijetunga) [reply ui/src/pages/AllStories.tsx:8]: If not we'll also need to add this logic in a separate pr

**Review by nimwijetunga — APPROVED** [review]
Works for me!

File: ui/src/pages/AllStories.tsx (hunk 3 above)
Comment (aspensantos) [reply ui/src/pages/AllStories.tsx:8]: yes when u click an entry in the table it opens the view story page for that entry

File: ui/src/pages/Upload.tsx (hunk 2 above)
Comment (aspensantos) [reply ui/src/pages/Upload.tsx:1]: Yeah for edit story we would do the same thing, I think the form is essentially the same it is just populated so we should be able to use the same logic to create the story object and pass that to the view story drawer

File: ui/src/pages/Upload.tsx (hunk 1 above)
Comment (aspensantos) [reply ui/src/pages/Upload.tsx]: done

**Pull request merged**